.storybook/manager.js

```diff
@@ -68,11 +68,7 @@ const SignInPanel = () => {
   });
 
   const emitProvider = loginState => {
-    if (Providers.globalProvider.state === ProviderState.SignedOut && Providers.globalProvider !== mockProvider) {
-      emit(SETPROVIDER_EVENT, { state: loginState, provider: mockProvider, name: 'MgtMockProvider' });
-    } else {
-      emit(SETPROVIDER_EVENT, { state: loginState, provider: msal2Provider, name: 'MgtMsal2Provider' });
-    }
+    emit(SETPROVIDER_EVENT, { state: loginState, provider: msal2Provider, name: 'MgtMsal2Provider' });
```

Comment:
This locks you into loggin in with a personal tenant only, is that the intention?

Reply:
Nope! Fixing this